ztp/kube-compare-reference/Makefile

```
@@ -24,4 +29,27 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1 helm
+
+
+.PHONY: convert
+convert: $(HELMCONVERTOR) helm
+	@echo "Converting reference files to Helm Charts."
+	@$(HELMCONVERTOR) -r ./metadata.yaml -n Chartv1 -v default_value.yaml
+	@echo "Rendering Helm Charts to CR files."
+	@./helm template renderedv1 ./Chartv1 --output-dir renderedv1
+
+$(HELMCONVERTOR):
```

By having the HELMCONVERTER executable as the target, we may end up running against older versions of the tool by accident.  This will ensure we run `go install...` every time and this will forcibly keep the executable up to date:
```suggestion
.PHONY: $(HELMCONVERTOR)
$(HELMCONVERTOR):
```